samples/spring-client-java-valueentity-counter/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>spring-client-javavalueentity-counter</artifactId>
```

Comment:
I think we should call it `java-valueentity-counter-spring-client` though. 

All samples start with `java-` or `scala-`. That makes it easier for us to navigate.